bld/namelist_files/namelist_definition.xml

```diff
@@ -1393,6 +1393,22 @@ scheme (shallow convection).
 Default: set by build-namelist.
 </entry>
 
+<entry id="gw_convect_dp_ml" type="char*16" category="gw_drag"
+       group="gw_drag_nl" valid_values="off,on,bothoff,bothon" >
+Whether or not to use the ML scheme for gravity waves produced by deep convection.
+   'off'     no
+   'on'      yes
+   'bothoff' run both but use default for simulation
+   'bothon'  run both but use ML for simulation
```

Comment:
"`bothoff`" and "`bothon`" imply, to me, that ML and default are either both off/on. Would it make sense to split this into two separate options?

One for selecting a method. And one for selecting whether it's a comparison run?

Maybe something like:

```
method = ML
method = default
Comparison = True (run both)
Comparison = False (run ML/default only)
```

If this was already decided in a team meeting though, feel free to ignore my suggestions.

Reply:
The subtlety here is that we may want to run both schemes, but can only use the output from one to update the timestep.
So they are not mutually exclusive binary variables, but rather a sequence of 4 options.
I *think* this is the easiest approach, but happy to discuss further.

Reply:
After some thought I think this is possible and cleaner for our current case.
I have updated the variables to logicals accordingly following a split.